packages/docs/verite/patterns/identifier.md

```diff
@@ -4,15 +4,13 @@ sidebar_position: 4
 
 # Identifier Methods
 
-Verifiable Credentials allow flexible methods for identifying the issuer, the subject, and holder. The identifier data type is a URI, which can be a web page, Solid profile, a decentralized identifier (DID), etc.
+Verifiable Credentials allow flexible methods for identifying the issuer, the subject, and holder in any given situation. The identifier data type is a URI, which can be a web page, Solid profile, a decentralized identifier (DID), etc. Most of the example code in the documentation and sample implementation use DIDs, but this is exemplary and equivalent functionality using other portable and user-centric identifier systems could be entirely conformant.
 
-The default Verite implementations use [did:key](https://w3c-ccg.github.io/did-method-key/) and [did:web](https://w3c-ccg.github.io/did-method-web/) for convenience and simplicity, but you should use the identifier method that is appropriate for your requirements.
+The Verite sample implementations use [`did:key`](https://w3c-ccg.github.io/did-method-key/) (a scheme for representing offline/offchain key material in a DID-compatible format based on [the `multicodec` registry](https://github.com/multiformats/multicodec/blob/master/table.csv)), and [`did:web`](https://w3c-ccg.github.io/did-method-web/) (a simplifed resolution mechanism for using a well-known DNS domain as root of trust for self-publishing DID documents) for convenience and simplicity.  You should use the identifier method that is appropriate for your requirements, however, assessing distinctly the privacy, durability, and interoperability/intelligibility requirements separately for issuers, holders, and verifiers.
 
-On the subject/holder side, we chose did:key because it requires no fees or blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups. This enables identity wallet applications to mimic the behavior of existing crypto wallets.
+On the subject/holder side, we chose to model holder-side cryptography in the samepl implementation using `did:key` because it requires no gas/onchain fees nor assumes any blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups (i.e., the DID documents can be derived from the DID itself deterministically). This enables identity wallet applications to mimic the behavior of existing crypto wallets.
```

Comment:
```suggestion
On the subject/holder side, we chose to model holder-side cryptography in the sample implementation using `did:key` because it requires no gas/onchain fees nor assumes any blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups (i.e., the DID documents can be derived from the DID itself deterministically). This enables identity wallet applications to mimic the behavior of existing crypto wallets.
```